dcos/dcos-ui
# DCOS-46808: Rerun flaky tests on CI

## Testing



1. Create a fresh branch based off of this
2. Edit one or multiple integration test files on that branch to have a `it.only` or `describe.only` and push these changes
3. Follow "Debugging flaky integration tests" in Contributing.md
4. See a successful run if you picked a non-flaky test case, otherwise it should be a failing run

## Trade-offs



- Stayed with everything written in bash, it would have been too much to move both files to JS and having one in JS would mean code duplication. As this script is not always in use I did not want to risk it.

## Dependencies



~This is based of #3500, so let's get that one merged first.~

## Screenshots
